src/main/java/rose/Storage.java

```diff
@@ -0,0 +1,102 @@
+package rose;
+
+import rose.task.Task;
+import rose.task.Todo;
+import rose.task.Event;
+import rose.task.Deadline;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
```

Comment:
Consider placing import statements from the same package together and leave a line between them.
This can make it easier to browse the imports when there are too many of them.
It is great that you did not use wildcard imports though 😄